### ISSUE TITLE
Create an example Basic page

### ISSUE THREAD

**Issue by TravisCarden** [phenaproxima/starshot-prototype#122]
We currently have one example each of the _Blog post_ and _Event_ content types (and a webform). It would be good to have an example of our only other one: _Basic page_.

**Comment by phenaproxima** [comment]
Yeah, this would be a cool idea, with a link in the main menu! Tagging as a good first issue if anyone has the time.
